requirements.txt

```
@@ -7,7 +7,7 @@ ckanext-scheming==3.0.0
 -e git+https://github.com/depositar/ckanext-geoview.git#egg=ckanext-geoview
 -e git+https://github.com/depositar/ckanext-wikidatakeyword.git#egg=ckanext-wikidatakeyword
 -e git+https://github.com/ckan/ckanext-dcat.git@v1.5.1#egg=ckanext-dcat
-git+https://github.com/ckan/ckanext-showcase.git@v1.5.2#egg=ckanext-showcase
+git+https://github.com/ckan/ckanext-showcase.git@v1.6.1#egg=ckanext-showcase
```

```suggestion
git+https://github.com/ckan/ckanext-showcase.git@v1.8.3#egg=ckanext-showcase
```
Let's go straight to the v1.8.3 since the it works without issues and the unit tests are passed in the depositar's CKAN environment.

---

Sure, let's upgrade directly to v1.8.3.